sw/device/tests/sim_dv/pwrmgr_deep_sleep_all_wake_ups.c

```diff
@@ -301,6 +303,10 @@ bool test_main(void) {
                  kTestWakeupSources[PWRMGR_PARAM_PINMUX_AON_USB_WKUP_REQ_IDX]
                      .wakeup_src)) {
     LOG_INFO("Woke up by source %d", PWRMGR_PARAM_PINMUX_AON_USB_WKUP_REQ_IDX);
+
```

Comment:
just to confirm this is behaving as expected, you only need to set this once right? not every time when you come out of low power?

Reply:
Hmm...Let me double check. I thought the latter was the case.
I will update in this thread.

Reply:
yeah let me know. If you need to do it every time, that would be a design bug i think.
Because @matutem and i discussed this before, the intent was for it to be a static setting.

Reply:
![image](https://user-images.githubusercontent.com/99843637/175614759-f56b1ddf-4ea4-47ab-8647-76101331c782.png)
As you see, usb goes off whenever lowpower event is triggerred.
Do I need to set
  CTRL.MAIN_PD_N & CTRL.USB_CLK_EN_LP to keep usb on?

Reply:
@tjaychen In case you miss this.

Reply:
yes that's right, when we trigger low power, the clock should go low.
But if you have the USB_CLK_EN_ACTIVE set to high, when the system comes out of low power, it should automatically turn the clock back on.  Is that not what you're seeing? 

Basically my thought is that you should be able to leave `USB_CLK_EN_ACTIVE` 1 at the beginning of the test.

Reply:
So, "whenever pwrmgr goes out of low power,  USB_CLK_EN_ACTIVE has to be set manually. " is
true statement?

Reply:
no that's what i'm saying. The behavior "should" be that you set USB_CLK_EN_ACTIVE to high once.
Then whenever the system is NOT in low power, it should keep the `usb_clk` alive.

Reply:
That is correct... unless the RTL ends up changing the pwrmgr CONTROL behavior for some subset of resets.